Iceberg-TipUI.package/IceTipExistingBranchPanel.class/instance/initializeBranchesList.st

```diff
@@ -1,7 +1,10 @@
 initialization
 initializeBranchesList
+	| dataSource |
 	branchesList widget
 		addColumn: (IceTipTableColumn newAction: #shortDescriptionWithDecoration);
-		dataSource: self newBranchListDataSource;
+		dataSource: (dataSource := self newBranchListDataSource);
 		bindKeyCombination: Character cr toAction: [ self accept ];
-		onAnnouncement: FTStrongSelectionChanged do: [ self accept ]
\ No newline at end of file
+		onAnnouncement: FTStrongSelectionChanged do: [ self accept ].
+
+	self model branchModels detect: #isHead ifFound: [ :head | branchesList widget selectRowIndex: (dataSource elements indexOf: head) ] ifNone: [ self model hasBranches ifTrue: [ branchesList widget selectFirstVisibleRow ] ]
```

Comment:
Maybe we should add #selectElement: to FTTableMorph but since we need to be compatible to Pharo 6.1 it should not be done here.

Reply:
I still do not understand why we do not just do:

```
branchesList widget selectRowIndex: (dataSource elements indexOf: self model defaultBranchSelection)
```

or something like that...